Python/Product/Profiling/Profiling/CompareReportsWindow.xaml

```diff
@@ -3,8 +3,9 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:ui="clr-namespace:Microsoft.PythonTools.Profiling"
+    xmlns:pt="clr-namespace:Microsoft.PythonTools"
```

Comment:
Can we get the strings resources in the Profiling namespace? Having the same class fullname may cause issues if both are referenced (e.g. in test assemblies)